solar_system.rb

```diff
@@ -0,0 +1,33 @@
+require 'pry'
+class SolarSystem
+  attr_reader :star_name, :planets
+  def initialize(star_name)
+    @star_name = star_name
+    @planets = []
+  end
+
+  def add_planet(planet)
+    @planets << planet
+  end
+
+  def list_planets
+    string = " "
+    i = 0
+    @planets.map do |planet|
```

Comment:
This isn't converting the array of planets into an array of strings, so `.each` is probably better than `.map`